Dockerfile

```diff
@@ -53,6 +53,13 @@ RUN curl --location https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
  && apt-get clean \
  && rm --recursive --force /var/lib/apt/lists/*
 
+# INSTALL DOCKER
+RUN apt update && curl -fsSL https://get.docker.com -o get-docker.sh && sh get-docker.sh \
+   && apt-get clean \
+   && rm --recursive --force /var/lib/apt/lists/* \
+   && rm get-docker.sh
+
+
```

Comment:
### Follow-up of https://github.com/iterative/cml/pull/861#discussion_r787766420

Why don't we leave our containers without Docker for now? If we wish, we can create a follow–up issue to discuss what we need and how to integrate it.

```suggestion
```

Reply:
@iterative/cml